pageserver/src/page_service.rs

```diff
@@ -672,6 +673,39 @@ impl postgres_backend::Handler for PageServerHandler {
                 }
             }
             pgb.write_message_noflush(&BeMessage::CommandComplete(b"SELECT 1"))?;
+        } else if query_string.starts_with("show ") {
```

Comment:
Do we really need it? What is the use case? It is already available through http api

Reply:
Hm, for some reason I thought that there is already a `get` endpoint for config. Personally, I'm leaning towards using http api as much as possible without duplicating functionality in postgres protocol. But if people find these helpers useful let's keep them